### PR TITLE
add routing rules from verify-gsp environment

### PR DESCRIPTION
**Pull request by chrisfarms**
We are preparing to wire up the verify-gsp environment/cluster to deliver alerts to alertmanager:

* Add rules to route alerts from `product: verify` + `deployment: gsp` to the re-gsp slack channel as a starting point for mapping alerts somewhere useful.
* Add a rule to forward on the dead man's switch alert `severity:constant` to the cronitor ping endpoint

### CODE REVIEW

**Comment by chrisfarms** [comment]
I do not know how to set the `cronitor/verify-gsp-url` "secret"? but I have a cronitor monitor/URL ready to rock

**Comment by philandstuff** [comment]
secrets are stored in alphagov/re-secrets/observe and pulled out using `terraform-provider-pass`. I can help insert into the secrets store if needed

**Comment by philandstuff** [comment]
can you add a line like this in the appropriate place:
```
    verify_gsp_cronitor        = "${data.pass_password.verify_gsp_cronitor.password}"
```

**Comment by philandstuff** [comment]
`cronitor/verify-gsp-url` has been added to the password store 👍

**Comment by chrisfarms** [comment]
> can you add a line like this in the appropriate place:

Ooops ... 👍 done

File: terraform/projects/infra-security-groups-production/main.tf [hunk 1]
```diff
@@ -79,6 +79,12 @@ module "infra-security-groups" {
 
     "35.177.182.230/32",
     "35.177.122.134/32",
+
+    # verify gsp
+    "35.176.76.151",
+
+    "18.130.183.50",
+    "18.130.183.50",
```
Comment (issyl0) [inline terraform/projects/infra-security-groups-production/main.tf]: This looks like a duplicate of line 86?

Comment (issyl0) [reply terraform/projects/infra-security-groups-production/main.tf]:
```suggestion

```

Comment (chrisfarms) [reply terraform/projects/infra-security-groups-production/main.tf]: 66% of the time... it works _everytime_

Comment (chrisfarms) [reply terraform/projects/infra-security-groups-production/main.tf]: fixed 😇

Comment (issyl0) [reply terraform/projects/infra-security-groups-production/main.tf]: :-D

**Pull request merged**